src/calendar/date/DaySelectorPopup.ts

```diff
@@ -131,15 +131,30 @@ export class DaySelectorPopup implements ModalComponent {
 		this.currentDate = incrementMonth(this.currentDate, forward ? 1 : -1)
 	}
 
+	// Sets the content div (.main-view) to inert, disabling the ability to be focused, this traps the
+	// focus to the popup, releasing it just when the popup is closed and the inert property removed.
+	private focusHandling(trap: boolean) {
```

Comment:
should we call it `enableTrapFocus` or something?

Reply:
> should we call it `enableTrapFocus` or something?

"enable" makes me think that it only enables it. Maybe ```turnTrapFocus(on: boolean)```?

Reply:
yeah sounds better!